utils/test_http_server/connect.py

```diff
@@ -4,7 +4,24 @@
 from ipaddress import ip_address
 
 #HOST = '192.168.1.111'
-PORT = 9000
+PORT = 8000
+HTTP_OK = "HTTP/1.1 200 OK\r\n\r\n"
+HTTP_LOWLVL_GCODE = "HTTP/1.1 200 OK\r\nContent-Type: text/x.gcode\r\nContent-Length: 6\r\nCommand-Id: 1254\r\n\r\nG28 XY"
+HTTP_RESPONSE_ACC = "POST /p/events HTTP/1.0\r\nPrinter-Token: 01234567899876543210\r\nContent-Type: application/json\r\n\r\n{\"event\":\"ACCEPTED\",\"command_id\":1254}"
```

Comment:
need to check and also handle situation when content-length is wrong (both number larger and smaller than actual)